### PR TITLE
add Arabic translations

### PR DESCRIPTION
**Pull request by jinhojang6**
Add Arabic translations translated by Daoud

Screen Shot 2019-12-18 at 1 51 30 AM

### CODE REVIEW

File: themes/navy/source/js/main.js [hunk 1]
```diff
@@ -77,264 +77,927 @@
                         }for (var u = "function" == typeof require && require, i = 0; i < t.length; i++) o(t[i]);return o;
                       }return r;
                     })()({ 1: [function (require, module, exports) {
-                        $(document).ready(function ($) {
```
Comment (jakubgs) [inline themes/navy/source/js/main.js]: This file is modified due to a bug in build process. Don't commit this.

**Comment by jinhojang6** [comment]
@jakubgs 

dropped the changes in main.js and package-lock.json

In that case, should we update the README.md as well?

https://github.com/status-im/status.im#testing-locally

**Comment by jinhojang6** [comment]
Please take a look at the added commit

**Comment by jakubgs** [comment]
Looks good: https://dev.status.im/ar/

**Pull request merged**